Library/ViewModels/CommentCellViewModel.swift

```diff
@@ -83,6 +96,14 @@ public final class CommentCellViewModel:
     self.replyButtonIsHidden = Signal.combineLatest(isLoggedOut, isNotABacker)
       .map { isLoggedOut, isNotABacker in isLoggedOut || isNotABacker }
 
+    // If both the replyButton and flagButton should be hidden, the entire stackview will be hidden too
+    self.bottomRowStackViewIsHidden = Signal.combineLatest(
+      self.flagButtonIsHidden.signal,
+      self.replyButtonIsHidden.signal
+    ).map { flagButtonIsHidden, replyButtonIsHidden in
+      flagButtonIsHidden && replyButtonIsHidden
+    }
```

Comment:
Assuming the replyButton and flagButton should both be hidden we can just hide the entire stack view. Otherwise we'll see a larger-than-needed (read: unpleasant) padding at the bottom of the cell.